bunsen-spark/src/main/java/com/cerner/bunsen/spark/Bundles.java

```diff
@@ -368,25 +412,47 @@ public BundleContainer call(Tuple2<String, String> fileContentTuple) throws Exce
       this.support = FhirConversionSupport.supportFor(fhirVersion);
     }
 
+    ToResourceRow(String resourceName,
+        String resourceTypeUrl,
+        FhirVersionEnum fhirVersion,
+        SparkRowConverter converter,
+        List<String> contained) {
+
+      this.resourceName = resourceName;
+      this.resourceTypeUrl = resourceTypeUrl;
+      this.fhirVersion = fhirVersion;
+
+      this.converter = converter;
+      this.support = FhirConversionSupport.supportFor(fhirVersion);
+      this.containedResourceTypeUrls = contained;
+    }
+
     private void writeObject(java.io.ObjectOutputStream stream) throws IOException {
 
       stream.defaultWriteObject();
 
       stream.writeUTF(resourceName);
       stream.writeUTF(resourceTypeUrl);
       stream.writeObject(fhirVersion);
+      stream.writeObject(containedResourceTypeUrls);
     }
 
     private void readObject(java.io.ObjectInputStream stream) throws IOException,
         ClassNotFoundException {
 
       this.resourceName = stream.readUTF();
       this.resourceTypeUrl = stream.readUTF();
-      this.fhirVersion =  (FhirVersionEnum) stream.readObject();
+      this.fhirVersion = (FhirVersionEnum) stream.readObject();
+      this.containedResourceTypeUrls = (List<String>) stream.readObject();
 
       FhirContext context = FhirContexts.contextFor(fhirVersion);
 
-      this.converter = SparkRowConverter.forResource(context, resourceTypeUrl);
+      if (this.containedResourceTypeUrls == null || this.containedResourceTypeUrls.size() == 0) {
```

Comment:
nit: can we use `this.containedResourceTypeUrls.isEmpty()` instead?

Reply:
Fixed f0d5fdcc5c60ac12ae32e0b907dfdea91ab08fd8